.github/workflows/publish_docker.yml

```diff
@@ -0,0 +1,57 @@
+name: Publish Docker image
+on:
+  release:
+    types: [published]
+
+env:
+  BUILD_TYPE: Release
+
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+        
+    - name: Install LibTiff
+      run: sudo apt install libtiff-dev
+      
+    - name: Configure CMake
+      run: cmake -B ${{github.workspace}}/build-4-linux -DCMAKE_BUILD_TYPE=${{env.BUILD_TYPE}} -DBUILD_CLI=ON
+
+    - name: Build
+      run: cmake --build ${{github.workspace}}//build-4-linux --config ${{env.BUILD_TYPE}} --parallel 2
+
+    - name: Create Version File
+      run: python setup.py --version| grep -v init > VERSION
+    
+    - name: Set variable
+      run: |
+        VER=$(cat VERSION)
+        echo "VERSION=$VER" >> $GITHUB_ENV
+
+    - name: Log in to Docker Hub
+      uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+    
+    - name: Extract metadata (tags, labels) for Docker
+      id: meta
+      uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+      with:
+        images: sameeul/testrepo
```

Comment:
Should we keep `sameeul/testrepo` until we test then change it to the appropriate one, or?

Reply:
let's use my cred and test. And then update to PolusAI cred